aegis-web/yo/app/scripts/services/ExtendedMetadataAPIService.js

```diff
@@ -40,48 +40,152 @@
 'use strict';
 
 angular.module('hopsWorksApp')
+  .factory('ExtendedMetadataAPIService', ['$http', function ($http) {
+    const EXTENDED_METADATA_PROJECT_ENDPOINT = 'https://bbc6.sics.se:8181/hopsworks-api/aegis-metadata/catalogues/';
+    const EXTENDED_METADATA_DATASET_ENDPOINT = 'https://bbc6.sics.se:8181/hopsworks-api/aegis-metadata/datasets/';
+    const EXTENDED_METADATA_DISTRIBUTION_ENDPOINT = 'https://bbc6.sics.se:8181/hopsworks-api/aegis-metadata/distributions';
+    const API_KEY = '';
```

Comment:
which api key?

Reply:
The API key is currently documented in the Fraunhofer FOKUS internal project wiki, which is why I wanted to avoid commiting it to this GitHub repo. I'm actually working on a way to use config files for different environments for the front-end, where the API key might be stored in the future. As far as I've seen, there's no existing solution for this in the project right now